app/controllers/events_controller.rb

```diff
@@ -0,0 +1,112 @@
+class EventsController < ApplicationController
+  def index
+    @upcomming_events = Event.upcomming_events
+    @previous_events = Event.previous_events
+  end
+
+  def new
+    @event = Event.new()
+  end
+
+  def create
+    if sign_in?
+      event = current_user.created_events.build(event_params)
```

Comment:
we should be building this with events `current_user.events.build(....)`

Reply:
As stated before, we are using `#created_events` to connect User with Event 😄